airflow/www/utils.py

```diff
@@ -546,20 +547,35 @@ def pygment_html_render(s, lexer=lexers.TextLexer):
     return highlight(s, lexer(), HtmlFormatter(linenos=True))
 
 
-def render(obj, lexer):
+def render(obj: Any, lexer: Lexer, handler: Optional[Callable[[Any], str]] = None):
     """Render a given Python object with a given Pygments lexer."""
-    out = ""
+
     if isinstance(obj, str):
-        out = Markup(pygment_html_render(obj, lexer))
-    elif isinstance(obj, (tuple, list)):
+        return Markup(pygment_html_render(obj, lexer))
+
+    if isinstance(obj, (tuple, list)):
```

Comment:
Why `if` instead of `elif` as in the old code?

Reply:
fixed, thanks :)